doc/source/cluster/metrics.md

```diff
@@ -80,6 +80,10 @@ Many developers are not on macOS's trusted list. Users can manually override thi
 
 See [these instructions](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac) for how to override the restriction and install or run the application.
 
+#### Loading Ray Prometheus configurations with Docker Compose
+In the Ray container, the symbolic link "/tmp/ray/session_latest/metrics" points to the latest active Ray session. However, Docker does not support the mounting of symbolic links on shared volumes and you may fail to load the Prometheus configuration files.
+
+To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```

Comment:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, the shared volume mount the shared volume on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```

Reply:
I think it should be
```
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, mount the shared volume on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```
There's an extra "the shared volume" before the newly suggested text
would help if you could change the suggestion , ill commit it.